tests/Common/Api/PublishedProduct/GetPublishedProductApiIntegration.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Akeneo\PimEnterprise\tests\Common\Api\PublishedProduct;
+
+/**
+ * @author    Olivier Soulet <olivier.soulet@akeneo.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class GetPublishedProductApiIntegration extends AbstractPublishedProductApiTestCase
```

Comment:
Put everything in common as 1.7 does not exist.

The rule is: put everything in common, if it fails, it's either a regression or a change that is not important/assumed (an order that change, variant_group property in products, etc).

Reply:
Done.